src/Plugin/QueueWorker/NotificationQueueWorker.php

```diff
@@ -154,10 +154,17 @@ private function yieldLoads(array $notifications): \Generator {
     foreach ($notifications as $notification) {
       /** @var \Lullabot\Mpx\DataService\Media\Media $mpx_media */
       $mpx_media = $notification->getNotification()->getEntry();
+      switch ($notification->getNotification()->getMethod()) {
+        case "delete":
+          $this->importer->unpublishItem($notification->getNotification()->getId(), $notification->getMediaType());
```

Comment:
I think this will be the id of the notification object, not the id of the media object. Does this need to be:

`$notification->getNotification()->getEntry()->getId()`?

Reply:
done.